sdv/metadata/visualization.py

```diff
@@ -105,10 +107,10 @@ def visualize_graph(nodes, edges, filepath=None):
     )
 
     for name, label in nodes.items():
-        digraph.node(name, label=graphviz.escape(label))
+        digraph.node(name, label=_replace_special_characters(label))
```

Comment:
have you tried using https://graphviz.readthedocs.io/en/stable/api.html#graphviz.nohtml

Reply:
Yes, it doesn't address the problem.

Reply:
I see. It seems like it works if you replace '>' with '\\>'. Then in the output graph the label looks correct 
<img width="268" alt="Screenshot 2023-11-08 at 9 51 51 PM" src="https://github.com/sdv-dev/SDV/assets/7800528/0a4d7650-9af0-4a1f-bae8-5ce5b54c1583">

Reply:
Just a note, this approach removes the backslash from the column name. So if a column has `\<` in it, the output will be `<`.

Reply:
I think that's ok since prior to this change if they had '\\>' in a label, it would only show '>' anyway.